chessticulate_api/models.py

```diff
@@ -110,7 +110,7 @@ class Game(Base):
     white: Mapped[int] = mapped_column(ForeignKey("users.id"), nullable=False)
     black: Mapped[int] = mapped_column(ForeignKey("users.id"), nullable=False)
     whomst: Mapped[int] = mapped_column(ForeignKey("users.id"), nullable=False)
-    winner: Mapped[int] = mapped_column(ForeignKey("users.id"), nullable=True)
+    winner: Mapped[str] = mapped_column(String, nullable=True)
     is_active: Mapped[bool] = mapped_column(
         Boolean, nullable=False, server_default=sql.true()
     )
```

Comment:
Can we not just do a join like we did in the past for [get_invitations()](https://github.com/Chessticulate/chessticulate-api/blob/main/chessticulate_api/crud.py#L165)? Seems better to me to just use IDs as the foreign keys, and then join whenever we need more info for whatever row that foreign key belongs to.

Reply:
Yeah this would mess with our normalization. I'm also realizing that this might be unnecessary all together. we are already doing a join with Games and Users to grab the usernames of each player. We can probably just display the username of the winner by checking winner_id == white_id ? white_username : black_username. should've thought of that before lol.